src/App.js

```diff
@@ -31,17 +32,39 @@ function App() {
     setTrickStep(trickStep + 1)
   };
 
-  return (
-      <div className='stacks'>  
-        <Stack stack={ initialStack } /> 
-        <Button value='initial' onClick={ handleClick }/>
-      
-        <Stack stack={ middleStack } />
-        <Button value='middle' onClick={ handleClick }/>
-      
-        <Stack stack={ lastStack } />
-        <Button value='last' onClick={ handleClick }/>
+  const handleReset = (FIRST_TRICK_1_STEP) => {
+    const deck = createDeck();
+    const shuffledDeck = shuffle(deck);
+    const deck21cards = shuffledDeck.slice(0, CARDS_TO_USE_TRICK_1);
+    const { stack1, stack2, stack3 } = sliceDeckInThree(deck21cards);
+
+    setInitialStack(stack1);
+    setMiddleStack(stack2);
+    setLastStack(stack3);
+    setTrickStep(FIRST_TRICK_1_STEP)
+  }
+
+  if(trickStep === FINAL_TRICK_1_STEP) {
+    return (
+      <div>
+        <ChosenCard card={middleStack[ELEVENTH_CARD]} />
+        <br />
+        <Button value={FIRST_TRICK_1_STEP} buttonText='Play again' onClick={ handleReset }/>
```

Comment:
Não precisa desse `value`, já que você sempre saberá qual é o estado inicial:
```suggestion
        <Button buttonText='Play again' onClick={ handleReset }/>
```